### ISSUE TITLE
Can't use translate directive on <title> element

### ISSUE THREAD

**Issue by mpiasta-ca** [rubenv/angular-gettext#264]
In this case:

```
   <title translate translate-context="Title">My website title</title>
```

It is rendered like:

```
<title translate="" translate-context="Title">
   <span class="ng-scope">My website title</span>
</title>
```

The unexpected `<span>` causes the `<title>` to no longer display in the browser.

If I use a translate filter, I can no longer set a context, and the title appears as `{{'My website title' | translate }}` until the JavaScript is executed (undesirable behavior).

**Comment by gabegorelick** [comment]
See https://github.com/rubenv/angular-gettext/issues/191.

**Comment by rubenv** [comment]
Should be fixed with Angular 1.6
